vividus/src/test/java/org/vividus/runner/BddScenariosCounterTests.java

```diff
@@ -0,0 +1,155 @@
+
+import java.util.Properties;
+
+import com.github.valfirst.jbehave.junit.monitoring.JUnitReportingRunner;
+
+import org.apache.commons.cli.UnrecognizedOptionException;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.Description;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.vividus.SystemOutTests;
+import org.vividus.configuration.BeanFactory;
+import org.vividus.configuration.Vividus;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({ Vividus.class, BeanFactory.class, BddScenariosCounter.class })
+@PowerMockIgnore({ "javax.*", "com.sun.*", "jdk.*", "org.xml.*", "org.w3c.*" })
+public class BddScenariosCounterTests extends SystemOutTests
+{
+    private static final String DEFAULT_STORY_LOCATION = "story";
+    private static final String DIR_VALUE = "story/bvt";
+    private static final String FORMATTER = "%5d | %s";
+    private static final String STORIES = "Stories";
+    private static final String SCENARIOS = "Scenarios";
+    private static final String SCENARIOS_WITH_EXAMPLES = "Scenarios with Examples";
+    private static final String PROPERTIES = "properties";
+    private static final String ROOT = "root description";
+    private static final String STORY = "story description";
+    private static final String SCENARIO = "scenario description";
+    private static final String EXAMPLE = "example description";
+    private static final String BEFORE = "before description";
+    private static final String RESOURCE_LOCATION = "bdd.story-loader.batch1.resource-location";
+
+    @Mock
+    private Properties properties;
+
+    @Mock
+    private JUnitReportingRunner runner;
+
+    @Before
+    public void before()
+    {
+        PowerMockito.mockStatic(Vividus.class);
+        PowerMockito.mockStatic(BeanFactory.class);
+    }
+
+    @Test
+    public void testCounterIgnoresDescriptionsWithMethodNames() throws Exception
+    {
+        stubPropertiesAndRunnerInvocations();
+
+        Description root = Description.createSuiteDescription(ROOT);
+        Description beforeStories = Description.createTestDescription(Object.class, BEFORE);
+        Description story = Description.createSuiteDescription(STORY);
+        Description beforeStory = Description.createTestDescription(Object.class, BEFORE);
+        Description scenario = Description.createSuiteDescription(SCENARIO);
+
+        root.addChild(beforeStories);
+        root.addChild(story);
+        story.addChild(beforeStory);
+        story.addChild(scenario);
+
+        when(runner.getDescription()).thenReturn(root);
+
+        BddScenariosCounter.main(new String[0]);
+
+        assertTrue(getOutput().contains(String.format(FORMATTER, 1, STORIES)));
```

Comment:
`String.format(FORMATTER, 1, STORIES)` -> replace with a plain string:
- there is no much sense to use `String.format` with constants only
- plain strings make unit tests more readable